src/test/java/com/heatheryou/httpserver/route/handler/GetHandlerTest.java

```diff
@@ -0,0 +1,26 @@
+package com.heatheryou.httpserver.route.handler;
+
+import com.heatheryou.httpserver.Request;
+import com.heatheryou.httpserver.Response;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class GetHandlerTest {
+    @Test
+    public void getHandlerReturnsStatusCode301IfUriIsRedirect() {
+        GetHandler handler = new GetHandler();
+        Request request = new Request("/redirect", "GET");
+        Response response = handler.handle(request);
+        String actual = response.getResponseLine();
+        String expected = String.join("\r\n", new String[]{
+                "HTTP/1.1 301 Moved Permanently",
+                "Location: http://0.0.0.0:5000/simple_get",
+                "Content-Length: 0",
+                "",
+                "",
+                ""
+        });
+        assertEquals(expected, actual);
+    }
+}
```

Comment:
Heck yeah feature tests! I'm not sure what direction or standards have been defined for coverage of features themselves, but with something like a server or API, I think it's neat to have a setup where you can test your actual server by running it and hitting the real feature that you just built.